apps/web/cypress/tests/integrations-list-modal.spec.ts

```diff
@@ -0,0 +1,959 @@
+
+    clickOnListRow('Novu In-App');
+
+    cy.getByTestId('update-provider-sidebar').should('be.visible');
+    cy.getByTestId('novu-in-app-frameworks').contains('React').click();
+
+    cy.getByTestId('update-provider-sidebar').contains('React integration guide');
+    cy.getByTestId('sidebar-back').should('be.visible');
+    cy.getByTestId('setup-timeline').should('be.visible');
+    cy.getByTestId('update-provider-sidebar-update').should('be.disabled').contains('Update');
+  });
+
+  it('should show the Novu Email integration sidebar', () => {
+    cy.intercept('GET', '*/integrations', async (req) => {
+      req.continue((res) => {
+        const {
+          body: { data },
+        } = res;
+        const [firstIntegration] = data;
+
+        res.body.data = [
+          ...data,
+          {
+            _id: EmailProviderIdEnum.Novu,
+            _environmentId: firstIntegration._environmentId,
+            providerId: EmailProviderIdEnum.Novu,
+            active: true,
+            channel: ChannelTypeEnum.EMAIL,
+            name: 'Novu Email',
+            identifier: EmailProviderIdEnum.Novu,
+          },
+        ];
+      });
+    }).as('getIntegrations');
+    cy.intercept('POST', '*/integrations', async () => {
+      await new Promise((resolve) => setTimeout(resolve, 1000));
+    }).as('createIntegration');
+    cy.intercept('GET', '*/integrations/email/limit', async () => {
+      await new Promise((resolve) => setTimeout(resolve, 200));
+    }).as('getNovuEmailLimit');
+    cy.intercept('DELETE', '*/integrations/*', async () => {
+      await new Promise((resolve) => setTimeout(resolve, 1000));
+    }).as('deleteIntegration');
+    cy.intercept('*/environments').as('getEnvironments');
+
+    navigateToGetStarted();
+
+    cy.wait('@getIntegrations');
+    cy.wait('@getEnvironments');
+
+    cy.getByTestId('sidebar-close').should('be.visible').click();
+
+    clickOnListRow('Novu Email');
+
+    cy.wait('@getNovuEmailLimit');
+    cy.getByTestId('update-provider-sidebar-novu').should('be.visible');
+    cy.window().then((win) => {
+      if (win.isDarkTheme) {
+        cy.getByTestId('update-provider-sidebar-novu').find(
+          'img[src="/static/images/providers/dark/square/novu-email.svg"]'
+        );
+        return;
+      }
+
+      cy.getByTestId('update-provider-sidebar-novu').find(
+        'img[src="/static/images/providers/light/square/novu-email.svg"]'
+      );
+    });
+    cy.getByTestId('provider-instance-channel').should('contain', 'Email');
+    cy.getByTestId('provider-instance-environment').should('contain', 'Development');
+    cy.getByTestId('update-provider-sidebar-novu').contains('Novu Email');
+    cy.getByTestId('update-provider-sidebar-novu').contains('Free');
+    cy.getByTestId('novu-in-app-limits').then((el) => {
+      expect(el.get(0).innerText).to.eq(
+        'Novu provider allows sending max 300 emails per month,\nto send more messages, configure a different provider'
+      );
+    });
```

Comment:
data-test-id should be email instead of in-app? Or just novu-provider-limits ?

Reply:
yes, thanks for finding this!